network.ts

```diff
@@ -31,16 +30,16 @@ export const proxies: TablelandNetworkConfig = {
   matic: "0x5c4e6A9e5C1e1BF445A062006faF19EA6c49aFeA",
   filecoin: "0x59EF8Bf2d6c102B4c42AEf9189e1a9F0ABfD652d",
   sepolia: "0xc50C62498448ACc8dBdE43DA77f8D5D2E2c7597D",
-  "optimism-goerli": "0xC72E8a7Be04f2469f8C2dB3F1BdF69A7D516aBbA",
+  "optimism-sepolia": "0x68A2f4423ad3bf5139Db563CF3bC80aA09ed7079",
   "arbitrum-sepolia": "0x223A74B8323914afDC3ff1e5005564dC17231d6e",
   maticmum: "0x4b48841d4b32C4650E4ABc117A03FE8B51f38F68",
   "filecoin-calibration": "0x030BCf3D50cad04c2e57391B12740982A9308621",
-  "optimism-goerli-staging": "0xfe79824f6E5894a3DD86908e637B7B4AF57eEE28",
   // localhost is a stand alone node
   localhost: "",
   // local-tableland implies that a validator is also running. the proxy address will always be
   // "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512" because of the order of contract deployment
   "local-tableland": "0xe7f1725e7734ce288f8367e1bb143e90bb3f0512",
+  "optimism-sepolia-staging": "0xd39a346e8299F4e3685f3D44215e0c9328e73439"
```

Comment:
@brunocalza new address for op sepolia staging (I'll remove this key once contract gets verified)